src/calendar/grid/index.tsx

```diff
@@ -40,8 +46,53 @@ export interface GridProps {
   onGridKeyDownHandler: (event: React.KeyboardEvent<HTMLElement>) => void;
 }
 
+interface GridCellProps extends TdHTMLAttributes<HTMLTableCellElement> {
+  disabledReason?: string;
+}
+
+const GridCell = forwardRef((props: GridCellProps, focusedDateRef: React.Ref<HTMLTableCellElement>) => {
+  const { disabledReason, ...rest } = props;
+  const isDisabledWithReason = !!disabledReason;
+  const { targetProps, descriptionEl } = useHiddenDescription(disabledReason);
+  const ref = useRef<HTMLTableCellElement>(null);
+  const [showTooltip, setShowTooltip] = useState(false);
+
+  return (
+    <td
+      ref={useMergeRefs(focusedDateRef, ref)}
+      {...(isDisabledWithReason ? targetProps : {})}
+      {...rest}
+      onFocus={() => {
+        setShowTooltip(true);
```

Comment:
did not handle parent `onFocus` event here (like for date range picker GridCell), because calendar component handles focus event via ref